fuzzydir.lua

```diff
@@ -64,15 +84,15 @@ function normalize(path)
         path = string.sub(path, 3, -1)
     end
     if ends_with(path, "/") or ends_with(path, "\\") then
-        path = string.sub(path, 1, -2)
+        path = string.sub(path, 1, -1)
```

Comment:
This change makes the statement entirely pointless. It used to remove the trailing slash as part of path normalization, but with this change the string remains unchanged - this function call reads as "take a substring from 'path' from the first character (including it) to the last character (including it)".

Reply:
> This change makes the statement entirely pointless. It used to remove the trailing slash as part of path normalization, but with this change the string remains unchanged - this function call reads as "take a substring from 'path' from the first character (including it) to the last character (including it)".

This change is used to fix repeated loading. Use this script for the following folder structure:
```
│   1.mkv
│   1.srt
└───anyname*
    │      1_en.srt
    └───anyname*
        │      1_zh.srt
```
Before: mpv had two 1.srt.
After: mpv only had one 1.srt.

Reply:
Well, allowing mpv to discover everything IS the intended script behavior. Further selection of specific subtitles should be done by mpv (using the slang/alang config properties to specify preferred language priorities) or manually (`j` is the default hotkey for cycling between subtitles).

And, repeating my previous comment, this change doesn't do anything besides breaking the path normalization. Try it out on a fresh script copy - add logging of the `normalized` variable right before the return statement in the `explode` function and check both variants. The only time you'll see a different behavior is when a path in `sub-file-paths` ends with a slash (`sub-file-paths=**/` for example) - the script won't explode anything as it won't see the globbing pattern at the end because the trailing slash wouldn't be removed.